.test_database_urls/postgres_12

```diff
@@ -1,2 +1,2 @@
 export TEST_DATABASE_URL="postgresql://postgres:prisma@localhost:5434"
-set -e TEST_SHADOW_DATABASE_URL
+unset SHADOW_DATABASE_URL
```

Comment:
Should we do this everywhere? I don't know if it's portable either, but unlike `set -e` it works on zsh.

Reply:
whatever works in bash/zsh/fish (the most used shells in this company)